auth/user_mgt.go

```diff
@@ -498,6 +499,47 @@ func (c *baseClient) GetUserByPhoneNumber(ctx context.Context, phone string) (*U
 	})
 }
 
+// GetUserByProviderID gets the user data for the user corresponding to a given provider id.
+//
+// See
+// [Retrieve user data](https://firebase.google.com/docs/auth/admin/manage-users#retrieve_user_data)
+// for code samples and detailed documentation.
+//
+// `providerID` indicates the provider, e.g. 'google.com' for the Google provider.
```

Comment:
Suggest "such as" instead of "e.g."

My sense is that if we want to say "For example," it should come after a hard stop like a period or semicolon after "provider."  Internal style guide: https://developers.google.com/style/abbreviations#dont-use

Reply:
Done.